### PR TITLE
fix: translation warnings

### PR DESCRIPTION
**Pull request by dlockhart**
I was attempting to reduce the number of translation warnings we get that comment on our PRs. Unfortunately in fixing some things, I noticed partially untranslated stuff so I blew those away with English terms... so in the short term we may have more warnings.

The main change here is that we were being warned against not using full sentences inside each `select` / `plural` case. While technically it's possible to have words before & after them, the reasoning here around making it easier for translators to figure what to do is sound. I updated all the ones that I could, but the one was super complex so I just did the English one and we'll need new translations.

### CODE REVIEW

File: lang/ar.js [hunk 1]
```diff
@@ -26,13 +26,13 @@ export default {
...
-	"components.form-element.input.number.rangeOverflow": "يجب أن يكون الرقم {maxExclusive, select, true {أقل من} false {أقل من أو مساويًا لـ}} {max}.",
-	"components.form-element.input.number.rangeUnderflow": "يجب أن يكون الرقم {minExclusive, select, true {أكبر من} false {أكبر من أو مساويًا لـ}} {min}.",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {Number must be less than {max}.} other {Number must be less than or equal to {max}.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {Number must be greater than {min}.} other {Number must be greater than or equal to {min}.}}",
```
Comment (dlockhart) [inline lang/ar.js:31]: Just reset all the Arabic ones as all my attempts to do this automatically with RTL pasted stuff in the wrong place.

File: lang/da.js [hunk 2]
```diff
@@ -26,13 +26,13 @@ export default {
...
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {Tal skal være mindre end {max}.} other {Tal skal være mindre end eller lig med {max}.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {Tal skal være større end {min}.} other {Tal skal være større end eller lig med {min}.}}",
 	"components.form-element.input.text.tooShort": "{label} skal være på mindst {minlength} tegn",
 	"components.form-element.input.url.typeMismatch": "URL-adresse er ikke gyldig",
 	"components.form-element.valueMissing": "{label} er påkrævet.",
-	"components.form-error-summary.errorSummary": "Der blev fundet {count, plural, one { was 1 error } other { were { count } errors }} i den information, du sendte",
```
Comment (dlockhart) [inline lang/da.js:35]: This had a mix of English and Danish, so sending it back to the drawing board.

File: lang/en.js [hunk 3]
```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} is invalid.",
 	"components.form-element.defaultFieldLabel": "Field",
 	"components.form-element.input.email.typeMismatch": "Email is not valid",
-	"components.form-element.input.number.rangeError": "Number must be {minExclusive, select, true {greater than} false {greater than or equal to}} {min} and {maxExclusive, select, true {less than} false {less than or equal to}} {max}.",
-	"components.form-element.input.number.rangeOverflow": "Number must be {maxExclusive, select, true {less than} false {less than or equal to}} {max}.",
-	"components.form-element.input.number.rangeUnderflow": "Number must be {minExclusive, select, true {greater than} false {greater than or equal to}} {min}.",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
```
Comment (dlockhart) [inline lang/en.js:29]: For `rangeError`, it was hard enough to do the English one so I'm sending all the other languages back for re-translation. The others were easier so I was mostly confident I could "fix" them in the other languages.

File: lang/fr-fr.js [hunk 4]
```diff
@@ -26,21 +26,21 @@ export default {
...
+	"components.form-error-summary.errorSummary": "{count, plural, one {Il y avait une erreur dans les informations que vous avez soumises.} other {Il y avait {count} erreurs dans les informations que vous avez soumises.}}",
 	"components.input-date-range.endDate": "Date de fin",
 	"components.input-date-range.errorBadInput": "{startLabel} doit être antérieur à {endLabel}",
 	"components.input-date-range.startDate": "Date de début",
 	"components.input-date-time-range-to.to": "à",
 	"components.input-date-time-range.endDate": "Date de fin",
 	"components.input-date-time-range.errorBadInput": "{startLabel} doit être antérieur à {endLabel}",
 	"components.input-date-time-range.startDate": "Date de début",
-	"components.input-date-time.date": "Date",
+	"components.input-date-time.date": "Date", // mfv-translated
```
Comment (dlockhart) [inline lang/fr-fr.js:43]: This is a new feature that @bearfriend added where we can indicate that it's actually translated even though the English and in this case French words are the same.

File: lang/nl.js [hunk 5]
```diff
@@ -19,16 +19,16 @@ export default {
 	"components.filter.clearDescriptionSingle": "Filters wissen",
 	"components.filter.loading": "Laden van filters",
 	"components.filter.filterCountDescription": "{number, plural, =0 {Geen filters toegepast.} one {1 filter toegepast.} other {{number} filters toegepast.}}",
-	"components.filter.filters": "Filters",
+	"components.filter.filters": "Filters", // mfv-translated
```
Comment (dlockhart) [inline lang/nl.js:22]: Google translate says the Dutch word for "filters" is "filters", so yay.

File: lang/nl.js [hunk 6]
```diff
@@ -19,16 +19,16 @@ export default {
...
 	"components.filter.noFilters": "Geen beschikbare filters",
 	"components.filter.searchResults": "{number, plural, =0 {Geen zoekresultaten} one {1 zoekresultaat} other {{number} zoekresultaten}}",
 	"components.filter.singleDimensionDescription": "Filter op {filterName}",
 	"components.form-element.defaultError": "{label} is ongeldig.",
 	"components.form-element.defaultFieldLabel": "Veld",
 	"components.form-element.input.email.typeMismatch": "E-mailadres is ongeldig",
-	"components.form-element.input.number.rangeError": "Getal moet {minExclusive, select, true {greater than} false {greater than or equal to}} {min} en {maxExclusive, select, true {less than} false {less than or equal to}} {max} zijn.",
-	"components.form-element.input.number.rangeOverflow": "Getal moet {maxExclusive, select, true {less than} false {less than or equal to}} {max} zijn.",
-	"components.form-element.input.number.rangeUnderflow": "Getal moet {minExclusive, select, true {greater than} false {greater than or equal to}} {min} zijn.",
```
Comment (dlockhart) [inline lang/nl.js:31]: Another case where there was a mix of English and Dutch so sending these back.

File: lang/cy.js [hunk 7]
```diff
@@ -26,13 +26,13 @@ export default {
...
-	"components.form-element.input.number.rangeOverflow": "Rhaid i'r nifer fod {maxExclusive, select, true {yn llai na} false {yn llai na neu’n hafal i}} {max}.",
-	"components.form-element.input.number.rangeUnderflow": "Rhaid i'r nifer fod {minExclusive, select, true {yn fwy na} false {yn fwy na neu'n hafal i}} {min}.",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {Rhaid i'r nifer fod yn llai na {max}.} other {Rhaid i'r nifer fod yn llai na neu’n hafal i {max}.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {Rhaid i'r nifer fod yn fwy na {min}.} other {yn fwy na neu'n hafal i {min}.}}",
```
Comment (margaree) [inline lang/cy.js:31]:
```suggestion
	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {Rhaid i'r nifer fod yn fwy na {min}.} other {Rhaid i'r nifer fod yn fwy na neu'n hafal i {min}.}}",
```

File: lang/hi.js [hunk 8]
```diff
@@ -26,13 +26,13 @@ export default {
...
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {संख्या को इससे कम {max} होना आवश्यक है।} other {संख्या को इससे कम या इसके बराबर {max} होना आवश्यक है।}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {संख्या को इससे बड़ा {min} होना आवश्यक है।} other {संख्या को इससे बड़ा है या इसके बराबर {min} होना आवश्यक है।}}",
 	"components.form-element.input.text.tooShort": "{label} कम से कम {minlength} वर्णों का होना चाहिए",
 	"components.form-element.input.url.typeMismatch": "URL मान्य नहीं है",
 	"components.form-element.valueMissing": "{label} आवश्यक है।",
-	"components.form-error-summary.errorSummary": "आपके द्वारा सबमिट की गई जानकारी में {count, plural, one { 1 त्रुटि } other { { count } त्रुटियाँ }} मिली थी",
+	"components.form-error-summary.errorSummary": "{count, plural, one {आपके द्वारा सबमिट की गई जानकारी में  1 त्रुटि मिली थी} other {आपके द्वारा सबमिट की गई जानकारी में {count} त्रुटियाँ मिली थी}}",
```
Comment (margaree) [inline lang/hi.js:35]:
```suggestion
	"components.form-error-summary.errorSummary": "{count, plural, one {आपके द्वारा सबमिट की गई जानकारी में 1 त्रुटि मिली थी} other {आपके द्वारा सबमिट की गई जानकारी में {count} त्रुटियाँ मिली थी}}",
```

**Review by dlockhart — COMMENTED** [review]
Good eye @margaree, thanks!

**Review by margaree — APPROVED** [review]

**Pull request merged**